railties/lib/rails/generators/rails/controller/controller_generator.rb

```diff
@@ -25,6 +26,18 @@ def add_routes
       end
 
       private
+        def parent
+          options[:parent]
+        end
+
+        def parent_class_name
+          if parent
+            parent
+          else
+            "ApplicationController"
+          end
```

Comment:
```suggestion
          parent ||  "ApplicationController"
```

Reply:
I think it will be a good idea to enable the relevant rubocop's check - https://docs.rubocop.org/rubocop/1.31/cops_style.html#styleredundantcondition

Reply:
I agree this would be better addressed by enabling the cop. Looks like there's only 3 offenses in all of Rails and I've introduced 2 of them. 😅 

Would you like to open a PR to introduce this @fatkodima?

Reply:
Good idea @fatkodima ! 👍

Reply:
instead of adding this method, could a `default: "ApplicationController"` be specified on the `class_option`?

Reply:
Yes, but I think a more sweeping change could be made to change the other generators too so they look the same, more or less. I'll open another PR to do that. It is likely better to state the default in the class opt so it shows up in the help text.